cpp/marc_tools/single_marc_convert_keywords.cc

```diff
@@ -61,55 +57,46 @@ void readInGndKeywords(MARC::Reader * const marc_reader, std::unordered_map<std:
             gnd_keywords->emplace(std::make_pair(record.getFirstSubfieldValue("150", 'g'), record.getControlNumber()));
         if (not record.getFirstSubfieldValue("150", 'x').empty())
             gnd_keywords->emplace(std::make_pair(record.getFirstSubfieldValue("150", 'x'), record.getControlNumber()));
-
-        const std::string controlNumber = record.getControlNumber();
-        const std::string keywordSubfield = record.getFirstSubfieldValue("150", 'a');
-        const std::string keywordSubfieldG = record.getFirstSubfieldValue("150", 'g');
-        const std::string keywordSubfieldX = record.getFirstSubfieldValue("150", 'x');
-
-       // if (not keywordSubfield.empty() or not keywordSubfieldG.empty() or not keywordSubfieldX.empty())
-       //     std::cout << controlNumber << " a: " << keywordSubfield << " g:  " << keywordSubfieldG << " x: " << keywordSubfieldX << "\n";
-
     }
 
-        LOG_INFO("Processed " + std::to_string(record_count) + " MARC record(s).");
+    LOG_INFO("Processed " + std::to_string(record_count) + " entries.");
```

Comment:
I think the "MARC record(s)" terminology is clearer because it is more specific.